src/components/share-buttons.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import {
+  faLinkedinIn,
+  faFacebookF,
+  faTwitter,
+} from '@fortawesome/free-brands-svg-icons';
+
+export default function ShearButtons() {
+  function openPopup(e, type) {
+    if (e.type === 'click' || e.keyCode === 13) {
+      e.preventDefault();
+      let url = '';
+      switch (type) {
+        case 'fb':
+          url = 'https://www.facebook.com/sharer/sharer.php?u=';
+          break;
+        case 'li':
+          url = 'https://www.linkedin.com/sharing/share-offsite/?url=';
+          break;
+        case 'twitter':
+          url = 'https://twitter.com/share?url=';
+          break;
+        default:
+          url = '';
+      }
+      /* eslint-env browser, node */
+      return window.open(url + window.location.href, '_blank');
+    }
+    return true;
+  }
+  return (
+    <div className="text-left mt-8 text-black">
+      <h3 className="font-900 font-title text-2xl mt-6 ">Share</h3>
+      <button
+        type="button"
+        onKeyDown={(e) => openPopup(e, 'fb')}
+        onClick={(e) => openPopup(e, 'fb')}
+      >
+        <FontAwesomeIcon icon={faFacebookF} />
+      </button>
```

Comment:
This is too complex. We can do this much simpler with an anchor.

```js
const currentUrl = window.location.href;
return (
  <ul>
    <li className="inline">
      <a
        href={`https://www.facebook.com/sharer/sharer.php?u=${currentUrl}`}
        rel="noopener"
        target="_blank">
        <FontAwesomeIcon icon={faFacebookF} />
      </a>
    </li>
  </ul>
);
```

About `rel="noopener"` you can [read this](https://mathiasbynens.github.io/rel-noopener/).

Reply:
thanks indeed better i did the refactoring to the new approach